lib/biokbase/Transform/script_utils.py

```diff
@@ -136,7 +136,7 @@ def parse_docs(docstring=None):
     remainder = script_details["Args"]
     argument_values = list()
     for k in reversed(argument_keys):
-        remainder, value = remainder.split(k)        
+        remainder, _, value = remainder.rpartition(k)       
```

Comment:
Is it possible to rename ```name``` field into something like ```mzml_file_name```? It will allow us to be independent in this PR from changes in core file like ```script_utils.py``` which is not so easy to get merging approval for.